toolbox/fdc3-conformance/Intents-Tests.md

```diff
@@ -0,0 +1,247 @@
+- `2.0-RaiseIntentFailTargetedAppResolve1`: Perform above test, but:
+  - Use `fdc3.raiseIntent("aTestingIntent", testContextY, {appId: "<A's appId>"})`.
+  - You should receive a JavaScript Error with the message `ResolveError.IntentDeliveryFailed`.
+- `2.0-RaiseIntentFailTargetedAppResolve2`: Perform above test, but:
+  - Use `fdc3.raiseIntent("aTestingIntent", testContextX, {appId: "NonExistentApp"})`.
+  - You should receive a JavaScript Error with the message `ResolveError.TargetAppUnavailable`.
+- `2.0-RaiseIntentFailTargetedAppResolve3`: Perform above test, but:
+  - Use `fdc3.raiseIntent("sharedTestingIntent2", testContextY, {appId: "<H's appId>"})`.
+  - You should receive a JavaScript Error with the message `ResolveError.IntentDeliveryFailed` (as this app is configured for the intent and context pair, but does not add any intent listeners).
+  - **Note:  Test will need an extended timeout to allow for this to be returned in time by the desktop agent, which will have a vendor-defined timeout.**
+- `2.0-RaiseIntentFailTargetedAppResolve4`: Perform above test, but: 
+  - `fdc3.raiseIntent("sharedTestingIntent2", testContextY, {appId: "<I's appId>"})`
+  - You should receive a JavaScript Error with the message `ResolveError.IntentDeliveryFailed` (as this app is configured for the intent and context pair, but adds intent listeners of the wrong type.
+  - **Note:  Test will need an extended timeout to allow for this to be returned in time by the desktop agent, which will have a vendor-defined timeout.**
+- `2.0-RaiseIntentFailTargetedAppInstanceResolve1`: Perform above test, but:
+  - First spawn an instance of App **A** and collect its `AppIdentifier` with `const appIdentifier = await fdc3.open({appId: "<A's appId>"})`.
+  - Then use `fdc3.raiseIntent("aTestingIntent", testContextY, appIdentifier)` to target that instance.  
+  - You should receive a JavaScript Error with the message `ResolveError.NoAppsFound` (since A doesn't support this context type).
+- `2.0-RaiseIntentFailTargetedAppInstanceResolve2`: Perform above test, but:
+  - Use `fdc3.raiseIntent("aTestingIntent", testContextX, {appId: "<A's appId>", instanceId "NonExistentInstanceId"})`.  
+  - You should receive a JavaScript Error with the message `ResolveError.TargetInstanceUnavailable`.
+
+### Raise Intent Result (void result)
+
+| App | Step           | Details                                                                                           |
+|-----|----------------|---------------------------------------------------------------------------------------------------|
+| Test   | 1. Raise          | `fdc3.raiseIntent("aTestingIntent", testContextX)`<br>starts app A.                       |
+| B       | 2. Receive Intent & Context | After starting up, A runs `fdc3.addIntentListener("aTestingIntent")` to register its listener.<br>It then receives `testContextX`, matching that sent by Test |
```

Comment:
Is there a mismatch here? App column says B (and Test) but the details are talking about app A?